contracts/VotingEscrow.vy

```diff
@@ -50,6 +50,8 @@ interface IVeYfiRewards:
     def rewardCheckpoint(_account: address) -> bool: nonpayable
     def queueNewRewards(_amount: uint256) -> bool: nonpayable
 
+interface Migrator:
```

Comment:
Question:

Should we have a `IMigrator.sol`?

Reply:
We will add it when we need to migrate.